src/components/home/ReverifiPlus/index.jsx

```diff
@@ -33,7 +33,7 @@ function ReverifiPlus() {
           Search our network and find your supporting team to complete the
           process
         </Paragraph>
-        <Button type="button">See More</Button>
+        <MoreButton type="button">See More </MoreButton>
```

Comment:
Since styled components/buttons names are following up with the same displayed text in them. [example](https://github.com/abukhadraop-or/reverifi/pull/40/files#diff-59995230de1e1c8263909276b14497b254cfcbd730f970589a2a5b9ac5e96f90R48).
```suggestion
        <SeeMoreButton type="button">See More </MoreButton>
```